www/src/pages/guidelines/logo.js

```diff
@@ -33,37 +32,65 @@ import PartnershipLockups from "!raw-loader!../../assets/guidelines/partnership-
 
 import ColorSwatch from "../../components/guidelines/color/card"
 
-import { Box, Button, Flex, Text } from "../../components/guidelines/system"
+import { Button, Text } from "../../components/guidelines/system"
+import { Box, Flex } from "theme-ui"
 import theme from "gatsby-design-tokens/dist/theme-gatsbyjs-org"
 import palette from "../../utils/guidelines/extend-palette-info"
 
-const List = styled(`ul`)`
-  margin-left: 0;
-  padding: 0;
-  list-style: none;
-`
+const List = ({ children }) => (
+  <ul
+    sx={{
+      marginLeft: 0,
+      padding: 0,
```

Comment:
Just as a FYI, not a change request — the `sx` prop allows you to use shorthand, e. g. `p` instead of `padding`, `ml` instead of `marginLeft`. See https://theme-ui.com/sx-prop/#theme-aware-properties and https://theme-ui.com/sx-prop/#margin-and-padding ✌️

Reply:
@fk Thanks for letting me know, I will update this in other PR's

Reply:
Of course! Thank you!! 🔥🔥🔥